composer.json

```
@@ -90,8 +90,8 @@
         "drupal/redirect": "^1",
         "drupal/responsive_favicons": "^2.0",
         "drupal/scheduler": "^2 || ^2.0@RC",
-        "drupal/search_api": "1.17 || ^1.19",
-        "drupal/search_api_solr": "^4.1",
+        "drupal/search_api": "^1.35",
+        "drupal/search_api_solr": "^4.3.3",
```

Same issue with search_api_solr. https://www.drupal.org/project/search_api_solr/releases/4.3.3 Works with Drupal: ^10.1 || ^11.0

I think we need to allow https://www.drupal.org/project/search_api_solr/releases/4.3.1

```suggestion
        "drupal/search_api_solr": "^4.3.1",
```